packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# 51.5.0 - March 18, 2021
+
+## Added
+
+#### salesforcedx-sobjects-faux-generator
+
+- Define static representation of subset sobjects @W-8912349@ ([PR #3044](https://github.com/forcedotcom/salesforcedx-vscode/pull/3044))
+
+#### salesforcedx-utils-vscode
+
+- Changes to libraryCommandletExecutor interface to support cancellation ([PR #3054](https://github.com/forcedotcom/salesforcedx-vscode/pull/3054))
```

Comment:
I wonder if we need to include this change since it doesn't have any immediate customer impact.. maybe we can categorize this as something like "under-the-hood" changes for upcoming features?

Reply:
Agreed, no need to add this one to the changelog

Reply:
I would consider this internal engineering - I think once we enable the cancellation logic for Apex and SDR that is when we would want to surface it. This doesn't have any value for anyone other than us yet.